### ISSUE TITLE
Display some effects of weariness on fight 

### ISSUE THREAD

**Issue by waveyl** [CleverRaven/Cataclysm-DDA#46284]
# Is your feature request related to a problem? Please describe.
Without weariness, I can estimate whether I can defeat monsters or not, then choose to fight or escape, because I know almost everything about myself. But now, I can’t do that because of the mysterious weariness, especially it’s changed after every attack.

# Describe the solution you'd like
If you think there are too may effects to display, just show the most important, speed modifier, and vague expressions for others. But for me, the more, the better. 

# Describe alternatives you've considered
Continue to encourage players to die, confuse them, and blow their confidence in fighting.

**Comment by l29ah** [comment]
Yes, it's a huge bummer that weariness effect is not displayed under the SPEED category in `@`.

**Comment by actual-nh** [comment]
After every attack? The weariness level should only be changing every 5 minutes at most.
